src/neo/Wallets/SQLite/UserWallet.cs

```diff
@@ -10,15 +11,19 @@
 using System.Reflection;
 using System.Security;
 using System.Security.Cryptography;
+using System.Text;
 
 namespace Neo.Wallets.SQLite
 {
     public class UserWallet : Wallet
     {
+        private static readonly byte[] Salt = Encoding.ASCII.GetBytes("NEO");
```

Comment:
Can this `Salt` be randomly generated when creating the wallet database?

Reply:
good suggestion!